BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/CoreEventSource.cs

```diff
@@ -9,8 +9,19 @@ internal sealed class CoreEventSource : EventSource
     {
         public static readonly CoreEventSource Log = new CoreEventSource();
 
+#if NETSTANDARD2_0
+        public EventCounter BreezeResponseTimeCounter;
+#endif
+
         private readonly ApplicationNameProvider nameProvider = new ApplicationNameProvider();
 
+#if NETSTANDARD2_0
```

Comment:
why restrict to just netstandard20? we should be able to use it in net45/net46 as well right?

Reply:
EventCounter class access is available starting NETSTANDARD2.0, older framework does not have support for it.

Reply:
Its available in .NET Framework 4.7.1.
The issue with doing logging only in Netstandard2.0 is : For a app running in .net framework 471 or 48, nuget restores the net46 target (as sdk dont have net471 target), not the netstandard20. So those apps wont get eventcounter, even though net471/net48 has support for it.

Its probably a separate task - to add net471 target.

Reply:
Should we change the condition to #if !NET45 && !NET46 && !NETSTANDARD1_3, so future version will be taken care automatically.

Reply:
As per our offline discussion, will stick to NETSTANDARD2.0.